modules/packages/Buffers.chpl

```diff
@@ -495,16 +495,16 @@ module Buffers {
      :arg len_bytes: how many bytes to append to the buffer
   */
   proc buffer.append(b:byteBuffer, skip_bytes:int(64) = 0, len_bytes:int(64) = b.len) throws {
-    var err:syserr = ENOERR;
+    var err:errorCode = ENOERR;
     on this.home {
       err = qbuffer_append(this._buf_internal, b._bytes_internal, skip_bytes, len_bytes);
     }
     if err then try ioerror(err, "in buffer.append");
   }
 
   pragma "no doc"
-  proc buffer.append(b:byteBuffer, skip_bytes:int(64) = 0, len_bytes:int(64) = b.len, out error:syserr) {
-    compilerWarning("'out error: syserr' pattern has been deprecated, use 'throws' function instead");
+  proc buffer.append(b:byteBuffer, skip_bytes:int(64) = 0, len_bytes:int(64) = b.len, out error:errorCode) {
```

Comment:
Do we still need to have overloads that take `out error:syserr`? Won't this approach remove those out from under users feet? (I'm not sure what the status is on syserr deprecation.)

Reply:
Good point. Maybe the `out error: syserr` versions can be removed? It looks like they've been deprecated since May 2018.

Reply:
Great! That works for me!